cirq-core/cirq/sim/simulator_base.py

```diff
@@ -268,7 +268,7 @@ def _run(
         for i in range(repetitions):
             all_step_results = self._core_iterator(
                 general_suffix,
-                sim_state=act_on_args.copy() if i < repetitions - 1 else act_on_args,
+                sim_state=act_on_args.copy(True) if i < repetitions - 1 else act_on_args,
```

Comment:
Prefer named arg style `.copy(reuse_buffer=True)`, for boolean args.

Reply:
I see. The code is changed